src/js/components/ServiceDetailConfigurationTab.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import ConfigurationView from './ConfigurationView';
+import Service from '../structs/Service';
+
+class ServiceConfigurationTab extends React.Component {
+
+  render() {
+    let {service} = this.props;
+    let {currentVersionID} = service.getVersionInfo();
```

Comment:
Do you envision a case where we might want to pass the ConfigurationView a version that is not the current one? If not, I'd suggest to just pass it the `service`.